code/addons/test/src/node/reporter.ts

```diff
@@ -61,8 +60,6 @@ const isVitest3OrLater = vitestVersion
   : false;
 
 export class StorybookReporter implements Reporter {
-  testStatusData: API_StatusUpdate = {};
```

Comment:
@valentinpalkovic can you confirm this change is safe? I was looking through the usages of the `API_StatusUpdate` type and found this, but I don't see `testStatusData` used anywhere in the monorepo.

Reply:
I don't remember. Let's remove it!